app/views/companies/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h2><%= @company.name %></h2>
+  <table class="table table-rating">
+    <tr>
+      <th>Company Info:</th>
+      <td>
+        <%= link_to @company.website, fix_url(@company.website) %><br>
```

Comment:
we should probably normalize the url on the model using `before_save`, rather than fixing it in the view